### PR TITLE
✨ Responsives & Button Wrapper

### LINKED ISSUE #102: Responsive
Make **maintainer portal** completely responsive.

### PR DESCRIPTION
**Pull request by shvam0000**
# Description

- Fixed Responsiveness of the portal
- Button Wrapper Component

Fixes #102 #103  #104  #101 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Aradhya-Tripathi — APPROVED** [review]
LGTM

File: client/client-portals/pages/index.tsx [hunk 1]
```diff
@@ -13,12 +13,14 @@ const IndexPage = () => {
...
+    } else {
+      Router.push("/maintainer");
     }
   }, [authContext]);
 
   return (
-    <div className="min-h-screen bg-base-blue flex flex-col justify-center items-center">
-      <Link href="/maintainer">
+    <div>
+      {/* <Link href="/maintainer">
```
Comment (sakshi-choudhary) [inline client/client-portals/pages/index.tsx]: Please remove commented code

File: client/client-portals/components/admin/dashboard/index.tsx [hunk 2]
```diff
@@ -55,129 +55,120 @@ const ProjectApplications = () => {
...
+                      <Tick />
+                    </span>
+                  ) : (
+                    <Button
+                      onClick={() => {
+                        setOpen(true);
+                        setProjId(data._id);
+                      }}
+                      btnStyle="secondary"
+                      // className="flex justify-center w-1/8 mx-auto mt-4 bg-green-400 p-2 font-bold text-white rounded-xl"
```
Comment (sakshi-choudhary) [inline client/client-portals/components/admin/dashboard/index.tsx:141]: Remove commented code

File: client/client-portals/components/admin/dashboard/index.tsx [hunk 3]
```diff
@@ -55,129 +55,120 @@ const ProjectApplications = () => {
...
-    <>
-      <Layout type="admin">
-        <div className="flex flex-col items-center justify-center">
-          <CSSLoader />
-        </div>
-      </Layout>
-      <div className="fixed bottom-0 w-full">
-        <Footer />
+    <Layout type="admin">
+      <div className="flex flex-col items-center justify-center items-center mt-52">
```
Comment (sakshi-choudhary) [inline client/client-portals/components/admin/dashboard/index.tsx:168]: `items-center` is repeated 2 times

**Review by sakshi-choudhary — CHANGES_REQUESTED** [review]
Decrease Buttons' height for both type of button.

**Comment by sakshi-choudhary** [comment]
**Approve** type of buttons color should be `bright green` instead of `base-teal`

**Comment by sakshi-choudhary** [comment]
Decrease `x-axis` margin in mobile view for tables.

**Comment by sakshi-choudhary** [comment]
**Pagination buttons** should be at bottom

**Review by sakshi-choudhary — APPROVED** [review]
✅

**Pull request merged**